support/config/deployment.go

```diff
@@ -105,6 +105,24 @@ func (c *DeploymentConfig) ApplyTo(deployment *appsv1.Deployment) {
 	}
 	deployment.Labels[ManagedByLabel] = "control-plane-operator"
 
+	// adding annotation for EmptyDir volume safe-eviction
+	if deployment.Spec.Template.ObjectMeta.Annotations == nil {
```

Comment:
can we include a unit test for ApplyTo that validates this new logic?

Reply:
Done